live-examples/js-examples/array-tolocalestring.html

```diff
@@ -1,8 +1,8 @@
 <pre>
-<code id="static-js">var array1 = [1, 'a', new Date()];
+<code id="static-js">var array1 = [1, 'a', new Date('21 Dec 1997 14:12:00 UTC')];
 var localeString = array1.toLocaleString('en', {timeZone: "UTC"});
 
 console.log(localeString);
-// expected output: "1,a,1/18/2018, 3:32:00 AM"
+// expected output: "1,a,21/12/1997, 19:42:00"
```

Comment:
It seems the result of _toLocaleString_ is different on different platforms. I am using Google Chrome
![screenshot from 2018-01-19 08-59-27](https://user-images.githubusercontent.com/18121502/35133605-22398f82-fcf7-11e7-9cc1-f77e8e35b3d6.png)

Reply:
Oh. I guess that's because Chrome doesn't support the `options` or `locales` parameters :(.

Then perhaps the best thing would be to omit those parameters, and just add a comment like:

```
// expected output: "1,a,12/21/1997, 2:12:00 PM",
// This assumes "en" locale and UTC timezone - your results may vary
```